R/runner.R

```diff
@@ -0,0 +1,130 @@
+##' Run MCMC chains in series (one after another).  This is the
+##' simplest chain runner, and the default used by [mcstate_sample()].
+##' It has nothing that can be configured (yet).
+##'
+##' @title Run MCMC chain in series
+##'
+##' @return A runner of class `mcstate_runner` that can be passed to
+##'   [mcstate_sample()]
+##'
+##' @export
+mcstate_runner_serial <- function() {
+  run <- function(pars, model, sampler, n_steps, rng) {
```

Comment:
As you know, I got confused about what exactly rng needed to be here, and how that defined the number of chains, so maybe a comment on that would be useful, unless it's going to be really obvious.